<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapGeneratorApp</class>
 <widget class="QWidget" name="MapGeneratorApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Disciples II генератор случайных сценариев</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="5,0,2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Настройки игры</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
       <item>
        <widget class="QLabel" name="gameFolderLabel">
         <property name="text">
          <string>Папка с игрой:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="gameFolderEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="gameFolderButton">
         <property name="text">
          <string>Выбрать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Настройки генератора</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,1,1">
       <item>
        <widget class="QLabel" name="scenarioTemplateLabel">
         <property name="text">
          <string>Шаблон сценария:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="scenarioTemplateEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scenarioTemplateButton">
         <property name="text">
          <string>Выбрать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scenarioTemplateButtonReload">
         <property name="text">
          <string>Обновить</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
       <item>
        <widget class="QLabel" name="seedLabel">
         <property name="text">
          <string>Зерно генерации:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="seedEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="sizeLabel">
         <property name="text">
          <string>Размер сценария:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="sizeButtons">
         <item>
          <widget class="QRadioButton" name="size48Button">
           <property name="text">
            <string>48</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="size72Button">
           <property name="text">
            <string>72</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="size96Button">
           <property name="text">
            <string>96</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="size120Button">
           <property name="text">
            <string>120</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="size144Button">
           <property name="text">
            <string>144</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,4">
       <item>
        <widget class="QLabel" name="roadsLabel">
         <property name="text">
          <string>% дорог:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="roadsSpinBox">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,4">
       <item>
        <widget class="QLabel" name="forestLabel">
         <property name="text">
          <string>% лесов:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="forestSpinBox">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="0,4">
       <item>
        <widget class="QLabel" name="goldLabel">
         <property name="text">
          <string>Стартовое золото:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="goldSpinBox">
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="generateButton">
         <property name="text">
          <string>Генерировать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveScenarioButtom">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="aboutLabel">
         <property name="text">
          <string>Автор mak, 2022</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="zonesLabel">
         <property name="text">
          <string>Зоны</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="zonesImage">
       <property name="minimumSize">
        <size>
         <width>288</width>
         <height>288</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>288</width>
         <height>288</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/images/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="contentsLabel">
         <property name="text">
          <string>Наполнение</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="contentsImage">
       <property name="minimumSize">
        <size>
         <width>288</width>
         <height>288</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>288</width>
         <height>288</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/images/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
